.github/workflows/deploy-macos-windows.yml

```
@@ -1,62 +1,74 @@
-# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
-
-name: Deploy macOS Windows
-
-on:
-  push:
-    branches:
-      - master
-  release:
-    types:
-      - published
-jobs:
-  build:
-
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [windows-latest, macOS-latest]
-        python-version: [3.5, 3.6, 3.7, 3.8] 
-
-    steps:
-    - uses: actions/checkout@v2
-    - run: |
-        git fetch --prune --unshallow
-        
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-        
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade -q pip pytest wheel setuptools twine
-        
-    - name: Test with pytest
-      run: |
-        pip install .
-        pytest
-        
-    - name: Build wheel
-      run: python setup.py -q bdist_wheel
-      
-    - uses: actions/upload-artifact@v2
-      with:
-        name: Upload as build artifacts
-        path: dist/*.whl
-        
-    - name: Publish package to PyPI
-      if: github.event.action == 'published'
-      env: 
-        TWINE_USERNAME: __token__
-        TWINE_PASSWORD: ${{ secrets.pypi_password }}
-      run: twine upload --skip-existing dist/*.whl
-
-    - name: Publish package to TestPyPI
-      if: github.repository == 'ultrajson/ultrajson'
-      env:
-        TWINE_USERNAME: __token__
-        TWINE_PASSWORD: ${{ secrets.test_pypi_password }}
-      run: |
-        twine upload  --repository-url https://test.pypi.org/legacy/ --skip-existing dist/*.whl
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Deploy macOS Windows manylinux

```

```suggestion
name: Deploy wheels
```
And let's rename the file `.github/workflows/deploy-wheels.yml`